tests/test_commands.py

```diff
@@ -140,6 +140,10 @@ def test_validation_failed_message(self):
             stdout=output,
         )
 
-        self.assertIn("user", output.getvalue())
-        self.assertIn("783", output.getvalue())
-        self.assertIn("does not exist", output.getvalue())
+        output_str = output.getvalue()
+        self.assertIn("user", output_str)
+        self.assertIn("783", output_str)
+        # newer Django (>5.1) changes the error message from "does not exist" to "is not a valid choice"
+        self.assertTrue(
```

Comment:
would something like the following be more self documenting?

```python
import django 

if django.VERSION <= (5,1):
   self.assertIn("does not exist", output.getvalue())
else:
   self.assertIn("is not a valid choice", output.getvalue())
```

Reply:
I had to make it a test for `< (5,2)` because `(5,1)` is not `<= (5, 1, 1, 'final', 0)` .  I suppose the alternative would be to take the first 2 elements of the tuple, but I think this is readable enough....